index.html

```diff
@@ -71,3 +71,171 @@ <h1 class="home-title">Hey There.<br/>I’m Ye Min.</h1>
+          </div>
+          <ul class="multi-btns">
+            <li class="multi-btn">CSS</li>
+            <li class="multi-btn">HTML</li>
+            <li class="multi-btn">Bootstrap</li>
+            <li class="multi-btn">Ruby</li>
+          </ul>
+          <button class="see-project-btn" href="#">See Project</button>
+        </div>
+      </div>
+      <div class="six-wrap">
+        <div class="six-box-wrap">
+          <div class="six-body">
+            <h3 class="six-title">Profesional Art Printing Data</h3>
+            <p class="six-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="six-btns">
+              <li class="six-btn">HTML</li>
+              <li class="six-btn">Bootstrap</li>
+              <li class="six-btn">Ruby</li>
+            </ul>
+            <button class="six-project-btn" href="#">See Project</button>
+          </div>
+        </div>
+        <div class="six-box-wrap">
+          <div class="six-body">
+            <h3 class="six-title">Profesional Art Printing Data</h3>
+            <p class="six-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="six-btns">
+              <li class="six-btn">HTML</li>
+              <li class="six-btn">Bootstrap</li>
+              <li class="six-btn">Ruby</li>
+            </ul>
+            <button class="six-project-btn" href="#">See Project</button>
+          </div>
+        </div>
+        <div class="six-box-wrap">
+          <div class="six-body">
+            <h3 class="six-title">Profesional Art Printing Data</h3>
+            <p class="six-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="six-btns">
+              <li class="six-btn">HTML</li>
+              <li class="six-btn">Bootstrap</li>
+              <li class="six-btn">Ruby</li>
+            </ul>
+            <button class="six-project-btn" href="#">See Project</button>
+          </div>
+        </div>
+        <div class="six-box-wrap">
+          <div class="six-body">
+            <h3 class="six-title">Profesional Art Printing Data</h3>
+            <p class="six-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="six-btns">
+              <li class="six-btn">HTML</li>
+              <li class="six-btn">Bootstrap</li>
+              <li class="six-btn">Ruby</li>
+            </ul>
+            <button class="six-project-btn" href="#">See Project</button>
+          </div>
+        </div>
```

Comment:
- [ ] You have made quite a load of efforts on this project but I think there's a few thing to double-check in order to make your work more design-perfect/project requirement.
  - At the moment, the project cards don't have a physical background. See the image below and please fix it to improve your UI:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/242790344-b1a81fae-1659-49fb-b924-5cdc0b56871b.png)|![image](https://user-images.githubusercontent.com/99715640/242790394-170b7937-e2e4-422b-b9bf-83cfe591c902.png)|

- [ ] There are several states for a button element e.g. `button:hover`, `button:active`, etc. and several designs have been added to the Figma design s that young developers like yourself can implement to make your look more appealing and aesthetic. Here is the button pallete, kindly implement those states on your buttons to improve user experience. :fire: 
![image](https://github.com/mryeminthu/Portfolio/assets/99715640/e8bdc98d-6556-47d4-9028-cae02634921a)